.github/workflows/action-test.yaml

```diff
@@ -0,0 +1,301 @@
+name: csv-to-csvw action
```

Comment:
Ah ok, I see, you've added a test to this repo. I'm wondering if we can keep this repo clean by not including them? 

Can we have a separate repo which is designed for testing in which we reference the branch of this repo containing the latest code? Or better yet, is there a test harness we can use for local testing of actions? 

Better yet, could we have that testing repo and some python tests which take the branch to test as input and set up the testing repo with files in various formats, commit + push them to the testing repo and verify the expected behaviour using the github API? This beginning to sound like a lot of work but there may be steps we could take towards it.

Or perhaps better is [nektos/act](https://github.com/nektos/act) which claims to allow you to run github actions locally. That may be worth looking in to.

Reply:
@robons I think we should do an investigation here to identify the most suitable approach for coming up with a test harness. I will create a separate ticket for this.

I have removed the test files from the repo.